indico/modules/rb/client/js/common/rooms/RoomDetailsModal.jsx

```diff
@@ -171,6 +186,16 @@ function RoomDetails({bookRoom, room, availability, attributes}) {
               trigger={<Icon name="info circle" className="legend-info-icon" />}
               content={<TimelineLegend labels={legendLabels} compact />}
             />
+            <Button
```

Comment:
could we render this button as an `<a />` with a suitable `href`, so people can middle-click it and open a new tab with the results?

Reply:
that's a good idea, I'll give it a try